src/containers/shared/utils.js

```diff
@@ -124,11 +124,12 @@ export const localizeNumber = (num, lang = 'en-US', options = {}) => {
   return new Intl.NumberFormat(lang, config).format(number);
 };
 
-export function formatPrice(number, lang = 'en-US', currency = 'USD', decimals = 4) {
```

Comment:
why would we ever pad with something other than `0`? if there's no need for it now, having the additional parameter is just noise. Let's get rid of it.

Reply:
It isnt what to pad with but how many places to pad.